From e2b5b450e7aac101d12149fdf0623106799933ff Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 2 Jul 2015 15:02:31 +0200
Subject: [PATCH] dracut-lib.sh:info() output info to stderr

otherwise it might collide with stdout redirection

(cherry picked from commit 57fb6e67de0cc58b9d188439b0ceb0d0d5dbbd6f)
---
 modules.d/99base/dracut-lib.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules.d/99base/dracut-lib.sh b/modules.d/99base/dracut-lib.sh
index b490e82..69f7079 100755
--- a/modules.d/99base/dracut-lib.sh
+++ b/modules.d/99base/dracut-lib.sh
@@ -51,7 +51,7 @@ if [ -z "$DRACUT_SYSTEMD" ]; then
         check_quiet
         echo "<30>dracut: $*" > /dev/kmsg
         [ "$DRACUT_QUIET" != "yes" ] && \
-            echo "dracut: $*"
+            echo "dracut: $*" >&2
     }
 
 else
@@ -61,7 +61,7 @@ else
     }
 
     info() {
-        echo "$*"
+        echo "$*" >&2
     }
 
 fi
